Utilities/VSOBuildScripts/postBuildScript.cmd

```diff
@@ -0,0 +1,43 @@
+if "%1" == "local" goto testlocal
```

Comment:
These build scripts should probably go into the Build folder along with the rest of the stuff.  Can the VSO build run powershell scripts?  We should probably keep the build scripts a consistent type.